src/main/java/com/skynest/utils/JsonTransformer.java

```diff
@@ -1,15 +1,19 @@
 package com.skynest.utils;
 
-import com.fasterxml.jackson.core.JsonProcessingException;
 import com.fasterxml.jackson.databind.ObjectMapper;
+import com.skynest.models.LoggedUserResponse;
+import io.restassured.response.Response;
+import io.restassured.response.ResponseBody;
 import lombok.AccessLevel;
 import lombok.NoArgsConstructor;
 
+import java.io.IOException;
+
 @NoArgsConstructor(access = AccessLevel.PRIVATE)
 public final class JsonTransformer {
     private static final ObjectMapper OBJECT_MAPPER = new ObjectMapper();
-
-    public static String objectToJson(Object payload) throws JsonProcessingException {
-        return OBJECT_MAPPER.writeValueAsString(payload);
+    public static LoggedUserResponse mapResponseToLoggedUserResponse(Response response) throws IOException {
```

Comment:
It would be a better long term solution to make this helper generic or you might find yourself writing a method for each class that you want to map to